charts/govuk-rails-app/templates/deployment.yaml

```diff
@@ -65,11 +65,13 @@ spec:
             failureThreshold: 2
             periodSeconds: 5
           {{- end }}
-          {{- if .Values.securityContext.appContainer }}
           securityContext:
-            allowPrivilegeEscalation: false
-            runAsUser: 1001
-            runAsGroup: 1001
+            allowPrivilegeEscalation: {{ .Values.securityContext.allowPrivilegeEscalation }}
+            runAsUser: {{ .Values.securityContext.runAsUser }}
+            runAsGroup: {{ .Values.securityContext.runAsGroup }}
```

Comment:
You probably want to apply same to Worker-deployment to keep both containers consistent?

Reply:
Whoops, good catch, thanks!